### PR TITLE
feat(widget-builder): Replace GSH with page filters

### PR DESCRIPTION
**Pull request by davidenwang**
Before:
image

After:
image

### CODE REVIEW

**Comment by priscilawebdev** [comment]
I noticed that the pageFilters are taking the whole width in dashboards. is this expected?

image

**Comment by priscilawebdev** [comment]
it environment item looks a bit odd on smaller devices. do we plan to improve it?

image

File: static/app/components/organizations/pageFilterBar.tsx [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
   display: flex;
   position: relative;
   border-radius: ${p => p.theme.borderRadius};
   height: ${p => p.theme.form.default.height}px;
+  ${p =>
+    p.condensed &&
+    `
+    max-width: 100%;
+    width: max-content;
+  `}
```
Comment (taylangocmen) [inline static/app/components/organizations/pageFilterBar.tsx:13]: nit
```suggestion
    css`
    max-width: 100%;
    width: max-content;
  `}
```

**Comment by davidenwang** [comment]
> I noticed that the _pageFilters_ is taking the whole width in dashboards. is this expected?
> 
> image

ooh definitely not expected, sorry this was fixed in https://github.com/getsentry/sentry/pull/33714

File: static/app/components/organizations/pageFilterBar.tsx (hunk 1 above)
Comment (davidenwang) [reply static/app/components/organizations/pageFilterBar.tsx]: ooh sorry this change was actually from a previous parent PR, I will make the change there though.

**Review by priscilawebdev — APPROVED** [review]
tested locally and it looks good to me

**Pull request merged**